seed.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+require('dotenv').config();
+const mongoose = require('mongoose');
+const Book = require('./modules/bookModel.js');
+
+async function seed() {
+  // mongoose.connect('mongodb+srv://aenyeart:mongolia28@cluster0.bg6da.mongodb.net/books?retryWrites=true&w=majority');
+  mongoose.connect(process.env.MONGO_DB_LIVE_URL);
+
+  await Book.create({
+    title: 'Consider Phlebas',
+    description: 'Science fiction book',
+    status: 'read',
+    email: 'eriksavage@fake-email.com',
+  }); 
```

Comment:
- We may need to use `save()` after we create books to seed the data. After you send me over email/user info you used to seed data (see Canvas note) I can take a look 👍

Reply:
- I looked into this and turns out I was behind on Mongoose reading :) create() pretty much does same thing as save, whew! :)